.travis.yml

```diff
@@ -54,13 +54,13 @@ before_install:
       source <(curl -SL https://git.io/fhPYW -o -)
     fi
 install:
-  - pip install tox codecov
+  - pip install tox
 
 script:
   - tox -e $TOX_ENV
 
 after_script:
-  - codecov
+  - tox -e publishcov
```

Comment:
do we need this ?

For me spinning a new virtualenv just to push the coverage is wasted time, especially since this code is designed to be executed in Travis-CI and GHA where we have a new fresh VM for each build.

Reply:
this makes everything consistent, and we can start using `tox --parallel-live` soon

Reply:
We can leave this change but a developer running tests on the local system should not push the code coverage results.

Reply:
The GITHUB_TOKEN is rolled every build so would be very tricky for someone to succeed in uploading this once the CODECOV_* Travis vars are revoked